src/sass/style.scss

```diff
@@ -143,6 +146,31 @@ section {
   }
 }
 
+nav {
```

Comment:
Most of your changes can be achieved by providing variables for top bar component. Check `_top-bar.scss` file.

Note that variables you override should be **before** component import statement.